tools/labs/templates/kernel_api/6-list-sync/list-sync.c

```diff
@@ -46,18 +46,12 @@ static struct task_info *task_info_find_pid(int pid)
 	struct list_head *p;
 	struct task_info *ti;
```

Comment:
@MathieuBordere thanks for the contribution. First observation is can you add the description from PR to the commit message. It looks more detailed.

Reply:
Ok, done